collective/easyform/profiles/uninstall/controlpanel.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <object name="portal_controlpanel">
- <configlet title="EasyForm" action_id="EasyForm" appId="EasyForm"
+ <configlet title="easyform" action_id="easyform" appId="easyform"
     category="Products" condition_expr=""
     remove="True">
   <permission>Manage portal</permission>
```

Comment:
You can remove this <permission> line. In uninstall profile you usually just have to provide an identifier and the `remove="True"` part. So in this case you could probably drop line 4 as well (`category="Products" condition_expr=""`) for the sake of compactness.